app/views/shared/_micropost_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_for @micropost do |f| %>
+<%= render "shared/error_messages", object: f.object %>
```

Comment:
check coding convention for this file, please. (indent wrong such as line 2, ...)